### PR TITLE
Enable debug test cases

### PR DESCRIPTION
**Pull request by praveennadarajah**
## Purpose
This PR will enable the following debug test cases.

- [x] MultiModuleBuildDebugTest
- [x] MultiModuleRunDebugTest
- [x] SingleBalFileRunDebugTest
- [x] MultiModuleTestDebugTest
- [x] CallStackDebugTest
- [x] ControlFlowDebugTest
- [x] LanguageConstructDebugTest
- [x] BallerinaBuildRemoteDebugTest
- [x] BallerinaRunRemoteDebugTest
- [x] BallerinaTestRemoteDebugTest

Fixes: https://github.com/ballerina-platform/ballerina-lang/issues/25326 https://github.com/ballerina-platform/ballerina-lang/issues/25592

### CODE REVIEW

File: tests/jballerina-debugger-integration-test/src/test/resources/project-based-tests/breakpoint-tests/src/languageConstruct/mainLangConstruct.bal [hunk 1]
```diff
@@ -1,9 +1,22 @@
-import ballerina/test;
-import ballerina/http;
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (NipunaRanasinghe) [inline tests/jballerina-debugger-integration-test/src/test/resources/project-based-tests/breakpoint-tests/src/languageConstruct/mainLangConstruct.bal:1]:
```suggestion
// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: tests/jballerina-debugger-integration-test/src/test/resources/project-based-tests/breakpoint-tests/src/worker/mainWorker.bal [hunk 2]
```diff
@@ -1,21 +1,27 @@
-import ballerina/io;
-import ballerina/runtime;
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (NipunaRanasinghe) [inline tests/jballerina-debugger-integration-test/src/test/resources/project-based-tests/breakpoint-tests/src/worker/mainWorker.bal:1]:
```suggestion
// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: tests/jballerina-debugger-integration-test/src/test/resources/project-based-tests/breakpoint-tests/src/languageConstruct/mainLangConstruct.bal (hunk 1 above)
Comment (hemikak) [inline tests/jballerina-debugger-integration-test/src/test/resources/project-based-tests/breakpoint-tests/src/languageConstruct/mainLangConstruct.bal:1]: 2020

File: tests/jballerina-debugger-integration-test/src/test/resources/project-based-tests/breakpoint-tests/src/worker/mainWorker.bal (hunk 2 above)
Comment (hemikak) [inline tests/jballerina-debugger-integration-test/src/test/resources/project-based-tests/breakpoint-tests/src/worker/mainWorker.bal:1]: 2020

**Review by NipunaRanasinghe — APPROVED** [review]

**Pull request merged**